decidim-core/app/validators/password_validator.rb

```diff
@@ -96,10 +105,10 @@ def email_included_in_password?
   end
 
   def domain_included_in_password?
-    return false unless record&.current_organization&.host
-    return true if value.include?(record.current_organization.host)
+    return false unless organization || organization.host
```

Comment:
I think you need to use `&&` here instead of `||`? Because of `unless`.

If `organization` is set and `organization.host` is `nil`, `organization || organization.host` will return `organization`.

E.g. try this in rails console:

```ruby
> organization = Decidim::Organization.new
> puts "TEST1" unless organization || organization.host
> puts "TEST2" unless organization && organization.host
```

Reply:
Wow yes, of course, I had a brain fart, I've changed it